### PR TITLE
Fix/reset crash confirmation

### PR DESCRIPTION
**Pull request by zhangcc01**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Add reset crash confirmation feature on iOS and macOS test app.

iOS after:
Screen Shot 2019-06-24 at 16 23 40

macOS after:
Screen Shot 2019-06-21 at 11 18 09 AM

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

**Comment by jwallra** [comment]
Could you add screenshots?  Also I think a toast would be a good solution for @annakocheshkova 's issue

File: SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift [hunk 1]
```diff
@@ -223,6 +224,23 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextVie
...
+    alert.messageText = "Clear crash user confirmation?"
+    alert.addButton(withTitle: "OK")
+    alert.addButton(withTitle: "Cancel")
+    alert.alertStyle = NSWarningAlertStyle
+    switch(alert.runModal()) {
+    case NSAlertFirstButtonReturn:
+        UserDefaults.standard.removeObject(forKey: kMSUserConfirmationKey)
+        break
+    case NSAlertSecondButtonReturn:
+        break
```
Comment (MatkovIvan) [inline SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift]: This case can be removed

Comment (zhangcc01) [reply SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift]: Yes, this case can be removed, default case include it .

Comment (zhangcc01) [reply SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift]: @MatkovIvan Yes, this case can be removed, default case include it .

**Pull request closed without merging**